pyttb/gcp/handles.py

```diff
@@ -84,17 +84,17 @@ def rayleigh(data: np.ndarray, model: np.ndarray) -> np.ndarray:
 
 def rayleigh_grad(data: np.ndarray, model: np.ndarray) -> np.ndarray:
     """Return gradient function for rayleigh distributions"""
-    return 2 / (model + EPS) - (np.pi / 2) * data**2 / (model + EPS) ** 3
+    return 2 / (model + EPS) - (np.pi / 2) * np.power(data, 2) / (model + EPS) ** 3
```

Comment:
Is there a reason to use np power over `**` I assumed ndarray implemented pow themselves. Same general question for no multiply etc on the proposed changes.

Reply:
Yes, when I tested it I got the following error (Python 3.10.8):
`TypeError: unsupported operand type(s) for ** or pow(): 'memoryview' and 'int'`
I got similar errors for the other binary operators that I changed.

Reply:
Ahh you casted the sptensor directly to double instead of to a dense tensor. `double()` vs `full()`. If you call `.data` on a dense tensor you get a numpy array, but if you call `.data` on a numpy array you get direct access to the underlying memory layout. Did you run the packaging tests locally on this? `mypy -p pyttb`. I assume that would point out the mixed up types.

Reply:
Here is where I had called .data assuming a tensor type resulting in numpy array as input to the function_handle https://github.com/jeremy-myers/pyttb/blob/402e2aafd5fe829c2ea9bb3079675942e8f51249/pyttb/gcp/fg.py#L87

Reply:
Ah! I just saw this. I opened a PR in the main repo. Can we take this conversation over there?

Reply:
Sounds good. I'll check over there for it tomorrow.